admin/templates/institutions/detail.html

```diff
@@ -50,6 +50,9 @@ <h4>Banner:</h4>
                     <button id="show-modify-form" class="btn btn-link" type="button">
                         Modify Institution
                     </button>
+                    <div class="col-md-12">
+                        <a class="btn btn-link" href={% url 'institutions:register_metrics_admin' institution.id %}>Register Moderator/Admin</a>
```

Comment:
I'd say "create Moderator/Admin" usually we only say register when making new accounts.